knowledge-base/actions/crowdin/github-action/action-security.yml

```diff
@@ -0,0 +1,3 @@
+name: 'crowdin-action'
```

Comment:
I think permission here would be:

`contents: write`, `to download translation into new branch`
`contents-if: ${{ contains(with, 'download_translations') && with['download_translations'] == 'true'}}`

`pull-requests: write`, `to create pr with downloaded translations`
`pull-requests-if: ${{ contains(with, 'create_pull_request') && with['create_pull_request'] == 'true'}}`

view [here](https://github.com/crowdin/github-action#supported-options), [here](https://github.com/crowdin/github-action/blob/3461273fc0576cdc442472e6b8759769bf6383d7/action.yml#L41)

@varunsh-coder please review this file.